torchvision/transforms/transforms.py

```diff
@@ -2014,3 +2014,51 @@ def forward(self, img):
 
     def __repr__(self):
         return self.__class__.__name__ + f"(p={self.p})"
+
+
+class ElasticDeformation(torch.nn.Module):
+    """Transform a tensor image with elastic deformations.
+    This transform does not support PIL Image.
+    Given alpha and sigma, it will generate displacement
+    vectors for all pixels based on random offsets. Alpha controls the strength
+    and sigma controls the smoothness of the displacements.
+    The displacements are added to an identity grid and the resulting grid is
+    used to grid_sample from the image.
+    Applications:
+        Randomly transforms the morphology of objects in images and produces a
+        see-through-water-like effect.
+    Args:
+        alpha: List [alpha H, alpha W] or (float), defines the magnitude of displacements.
+        sigma: List [sigma H, sigma W] or (float) defining the smoothness of displacements.
+        interpolation: str saying either NEAREST, BILINEAR or BICUBIC. Defining the interpolation used in grid_sample.
+        fill: Pixel fill value for the area outside the transformed image. Default is 0.
+    """
+
+    def __init__(self, alpha, sigma, interpolation="bilinear", fill=0):
+        super().__init__()
+        if isinstance(alpha, list) and len(alpha) != 2:
+            raise ValueError("alpha should be a scalar or a list of two. Got a list of" f"{len(alpha)}.")
+
+        if isinstance(sigma, list) and len(sigma) != 2:
+            raise ValueError("sigma should be a scalar or a list of two. Got a list of" f"{len(sigma)}.")
+
+        self.alpha = alpha
+        self.sigma = sigma
+        self.interpolation = interpolation
+        self.fill = fill
+
+    def forward(self, tensor: Tensor) -> Tensor:
+        """
+        Args:
+            tensor (Tensor): Tensor image to be transformed.
+
+        Returns:
+            Tensor: Transformed image.
+        """
+        return elastic_deformation(tensor, self.control_point_spacing, self.sigma, self.interpolation, self.fill)
```

Comment:
Let's make the code same as other transforms:
```python
def forward(self, img):
    return F.elastic_deformation(img, ...)
```

For example, https://github.com/pytorch/vision/blob/fc52e86fa34413d35713ce264255a9f00902b2e2/torchvision/transforms/transforms.py#L1492-L1510